packages/legend-art/style/base/_common.scss

```diff
@@ -129,3 +142,34 @@
     border-radius: 0.2rem;
   }
 }
+
+.dnd__entry__container {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+// NOTE: We have a mask placeholder for PanelDnDEntry components
```

Comment:
TODO: let's remove this comment, this is actually better explained when we're in the component code